android/cli/commands/_buildModule.js

```diff
@@ -1758,8 +1790,7 @@ AndroidModuleBuilder.prototype.verifyBuildArch = function (next) {
 AndroidModuleBuilder.prototype.packageZip = function (next) {
 	this.logger.info(__('Packaging the module'));
 
-	fs.existsSync(this.distDir) || wrench.rmdirSyncRecursive(this.distDir);
-	wrench.mkdirSyncRecursive(this.distDir);
+	fs.ensureDirSync(this.distDir);
```

Comment:
This probably needs to be `fs.emptyDirSync`. The old check does not make sense though, because it would only try to remove the dist dir if it does not exists?!

Reply:
It was probably supposed to be an `&&`. I think Chris B uses this shorthand a lot to blow away and recreate a directory.